### DISCUSSION TITLE
Δήλωση πρόθεσης προαιρετικής παρουσίασης

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#93] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ερευνητικού άρθρου με την ανάρτηση  τουλάχιστον μια μέρα πριν την προθεσμία, δλδ **Παρασκευή 20 Οκτωβρίου** των παρακάτω στοιχείων:

1. της βιβλιογραφικής αναφοράς του άρθρου
2. συνδέσμου προς το άρθρο
3. του τελικού αρχείου παρουσίασης

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ1 δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη ΟΣΣ

**Comment by std133524** [comment]
Καλησπέρα,

θα ήθελα να δηλώσω την προαιρετική παρουσίαση

1. Teachable Machine: Approachable Web-Based Tool for Exploring Machine Learning Classification
2. [https://dl.acm.org/doi/abs/10.1145/3334480.3382839](https://dl.acm.org/doi/abs/10.1145/3334480.3382839)
3. [Teachable Machine.pdf](https://github.com/std133524/g1/blob/main/Teachable%20Machine.pdf)

**Comment by Asinapis13** [comment]
Καλησπέρα,

Θα ήθελα να δηλώσω την προαιρετική παρουσίαση


1. Supporting Social Interactions with an Expressive Heart Rate Sharing Application. Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technologies.
2. https://dl.acm.org/doi/epdf/10.1145/3130943
3.  https://github.com/Asinapis13/PRESENTATION.github.io/blob/main/SDY_60_1ERG_PRESENTATION_SINAPIS_AGGELOS.pdf

**Comment by std159312** [comment]
Καλησπέρα,

Θα ήθελα να δηλώσω την προαιρετική παρουσίαση

1. User Perceptions of Smart Home IoT Privacy
2. https://doi.org/10.1145/3274469
3. [Smart Home IoT Privacy.pdf](https://github.com/sdy60/syllabus/files/13187195/Smart.Home.IoT.Privacy.pdf)

**Comment by iparthenios** [comment]
Καλησπέρα!
Εδώ δηλώνω την πρόθεσή μου για την προαιρετική παρουσίαση:
1. Hand Gesture Recognition for Human Computer Interaction
2. https://www.sciencedirect.com/science/article/pii/S1877050917319130
3. Η παρουσίαση θα αναρτηθεί αργά απόψε.

**Reply by iparthenios** [reply]
[Hand Gesture Recognition for Human Computer Interaction.pdf](https://github.com/sdy60/syllabus/files/13183366/Hand.Gesture.Recognition.for.Human.Computer.Interaction.pdf)

**Comment by IoannisTheodoropoulos** [comment]
Καλησπέρα σας,
Θα ήθελα να δηλώσω κι εγώ τη προαιρετική παρουσίαση:

- Hoax News Detection on Social Media: A Survey
- https://doi.org/10.1109/INAPR.2018.8627053
- [Hoax News Detection.pdf](https://github.com/sdy60/syllabus/files/13181416/Hoax.News.Detection.pdf)

**Comment by Konssven** [comment]
Καλημέρα, 
Θα ήθελα και εγώ να δηλώσω τη πρόθεση μου για την προαιρετική παρουσίαση: 

1. Understanding the Long-Term Use of Smart Speaker Assistants
2. https://doi.org/10.1145/3264901 
3. [SDY60_1ERG_SVENTZOURIS_Part_D.pdf](https://github.com/sdy60/syllabus/files/13183934/SDY60_1ERG_SVENTZOURIS_Part_D.pdf)
